content/client-lib-development-guide/features.textile

```diff
@@ -73,9 +73,10 @@ h3(#restclient). RestClient
 ** @(RSC14c)@ When a @token@ or @tokenDetails@ is used to instance the library, and no means to renew the token is provided (either an API key, @authCallback@ or @authUrl@), if the server responds with a token error (401 HTTP status code and an Ably error value @40140 <= @code < 40150@), then the client library should indicate an error and not retry the request
 ** @(RSC14d)@ When the client does have a means to renew the token automatically, and the token has expired or the server has responded with a token error (@statusCode@ value of 401 and error @code@ value in the range @40140 <= @code < 40150@), then the client should automatically make a single attempt to reissue the token and resend the request using the new token. If the token creation failed or the subsequent request with the new token failed due to a token error, then the request should result in an error
 * @(RSC15)@ Host Fallback
-** @(RSC15a)@ All HTTP requests sent to Ably are fault-tolerant and will fallback to an alternative host name if the default host endpoint is unreachable or indicates that it is unserviceable with a 500 to 504 status code. This ensures that a client library is able to work around routing or other problems with the user's closest data center. For example, if a @POST@ request to @rest.ably.io@ fails because the default endpoint is unreachable or unserviceable, then the @POST@ request should be retried again against the fallback hosts in attempt to find an alternate healthy datacenter to service the request. The fallback hosts are @[a-e].ably-realtime.com@ and should be attempted in random order, with a maximum number of fallback attempts being the configured @httpMaxRetryCount@ (see "@TO3l5@":#TO3l5)
-** @(RSC15b)@ This fallback behaviour only applies to the default @rest.ably.io@ endpoint; if either @host@ or @environment@ options are set, no fallback host is used
-** @(RSC15d)@ Automatic retries to fallback hosts will not occur for any HTTP status codes not in the range 500 to 504. For example, if the credentials are invalid (a 401 status code) or the maximum message length has been exceeded, then resending the request to an alternative data center will not fix the problem and will only increase the load on other data centers. See the "complete list of Ably error codes":https://github.com/ably/ably-common/blob/master/protocol/errors.json#L11 and a "discussion in the Java library about this":https://github.com/ably/ably-java/issues/14
+** @(RSC15b)@ The fallback behaviour described below only applies when the default @rest.ably.io@ endpoint is being used and has not been overriden, see "RSC11":#RSC11. If host fallback is not supported, failing HTTP requests result in an error immediately and are not retried
```

Comment:
This needs to be clear that failing requests do not result in an alternative host being tried; there may still be a retry with the same host for other reasons (eg after token renewal).